docs/features.md

```diff
@@ -194,6 +202,14 @@ async def request_headers():
     }
 ```
 
+```python
+@app.get("/response_headers")
```

Comment:
We should have similar endpoint for `response_header` as for `request_header`?

Reply:
@ParthS007 , we already have one here(https://github.com/sansyrox/robyn/pull/410/files#diff-aff15ced1fe37506f41f2e1d93a4d8312529259f7803f8634ffb5eac4de47823R196)

Is this the same one that you're suggesting?